Makefile

```diff
@@ -19,13 +19,21 @@ NODE ?= node
 NPM ?= npm
 BUNDLER ?= $(BIN)/bundler
 I18N_CALYPSO ?= $(NODE_BIN)/i18n-calypso
-SASS ?= $(NODE_BIN)/node-sass --include-path 'client'
+SASS ?= $(NODE_BIN)/node-sass --include-path 'client' --include-path 'build'
 RTLCSS ?= $(NODE_BIN)/rtlcss
 AUTOPREFIXER ?= $(NODE_BIN)/postcss -r --use autoprefixer --autoprefixer.browsers "last 2 versions, > 1%, Safari >= 8, iOS >= 8, Firefox ESR, Opera 12.1"
 RECORD_ENV ?= $(BIN)/record-env
 ALL_DEVDOCS_JS ?= server/devdocs/bin/generate-devdocs-index
 COMPONENTS_USAGE_STATS_JS ?= server/devdocs/bin/generate-components-usage-stats.js
 
+# hash command
```

Comment:
Can you explain what the hash file is used for? And should we assume that one of `md5` and `md5sum` will be available on all platforms we care to support?